test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13623,7 +13709,7 @@ def run(self):
             self.job.interactive_handle = _p
             time.sleep(_st)
             expstr = "qsub: waiting for job "
-            expstr += "(?P<jobid>\d+.[0-9A-Za-z.-]+) to start"
+            expstr += "(?P<jobid>\d+.[0-9A-Za-z-]+) to start"
```

Comment:
I think this was a conflict , we did not resolve. Will keep in mind to include the change in next commit.